examples/README.md

```diff
@@ -0,0 +1,8 @@
+# Examples
+Example package contains a collection of snap examples and [videos](https://github.com/intelsdi-x/snap/blob/master/examples/videos.md) that demonstrate the snap libraries and usages.
+
+* [configs](https://github.com/intelsdi-x/snap/tree/master/examples/configs) is an example of the global configuration file that powers your plugins
```

Comment:
You can use relative pathing here:

```md
[configs](./configs)
```

Same for the Riemann and tasks examples.